### PR TITLE
Refactors the examine proc to return a list of strings

### PR DESCRIPTION
**Pull request by farie82**
**What does this PR do:**
Refactors examine() to return a list of strings and examinate to make use of this.
This will make morphs also look like their target reliably.

Cult constructs now show their examine text when you're next to it. Instead of on top of it.

I've tested this on a few things. The components as well. All looks fine to me

**Changelog:**
:cl:
fix: Morphs examine text now stays true to the humanoid target they choose.
tweak: Cult constructs now show their examine text when you're next to it. Instead of on top of it.
/:cl:

### CODE REVIEW

**Comment by farie82** [comment]
> This is a port, is it not?

Idea comes from https://github.com/tgstation/tgstation/pull/44636

File: code/game/objects/items/weapons/pneumaticCannon.dm [hunk 1]
```diff
@@ -22,16 +22,14 @@
 	return ..()
 
 /obj/item/pneumatic_cannon/examine(mob/user)
-	..()
+	. = ..()
 	if(!in_range(user, src))
-		to_chat(user, "<span class='notice'>You'll need to get closer to see any more.</span>")
-		return
-	for(var/obj/item/I in loadedItems)
-		spawn(0)
```
Comment (farie82) [inline code/game/objects/items/weapons/pneumaticCannon.dm]: This spawn was probably here to show the items on the bottom of the text. No clue why it was done like this but hey

File: code/game/atoms.dm [hunk 2]
```diff
@@ -272,7 +272,7 @@
 
 
 //All atoms
-/atom/proc/examine(mob/user, var/distance = -1, var/infix = "", var/suffix = "")
+/atom/proc/examine(mob/user, var/infix = "", var/suffix = "")
```
Comment (variableundefined) [inline code/game/atoms.dm]: No implied var please

File: code/game/gamemodes/wizard/soulstone.dm [hunk 3]
```diff
@@ -181,13 +181,13 @@
...
 /obj/structure/constructshell/examine(mob/user)
-	if(..(user, 0))
-		if(iscultist(user) || iswizard(user) || user.stat == DEAD)
-			to_chat(user, "<span class='cult'>A construct shell, used to house bound souls from a soulstone.</span>")
-			to_chat(user, "<span class='cult'>Placing a soulstone with a soul into this shell allows you to produce your choice of the following:</span>")
-			to_chat(user, "<span class='cult'>An <b>Artificer</b>, which can produce <b>more shells and soulstones</b>, as well as fortifications.</span>")
-			to_chat(user, "<span class='cult'>A <b>Wraith</b>, which does high damage and can jaunt through walls, though it is quite fragile.</span>")
-			to_chat(user, "<span class='cult'>A <b>Juggernaut</b>, which is very hard to kill and can produce temporary walls, but is slow.</span>")
+	. = ..()
+	if(in_range(user, src) && (iscultist(user) || iswizard(user) || user.stat == DEAD))
```
Comment (variableundefined) [inline code/game/gamemodes/wizard/soulstone.dm]: Undocumented change. Should include in CL & PR Description

File: code/game/atoms.dm (hunk 2 above)
Comment (farie82) [reply code/game/atoms.dm]: removed in the newest commit

File: code/game/gamemodes/wizard/soulstone.dm (hunk 3 above)
Comment (farie82) [reply code/game/gamemodes/wizard/soulstone.dm]: Changed in the description and CL

**Pull request merged**